README.md

```diff
@@ -27,146 +27,85 @@ https://unpkg.com/mmdynamo@0.2.0/dist/mmdynamo.cjs.js
 ## Initializing dynamodb
 
 ```js
-import { DynamoDB } from "mmdynamo";
-
-const client = new DynamoDB({
-  region: "us-east-2",
-  accessKeyId: "",
-  secretAccessKey: "",
-  service: "dynamodb",
-  endpoint: "http://dynamodb.us-east-2.amazonaws.com",
+import  Client  from "mmdynamo";
+
+const client = new Client({
+  agent: "fetch",
+  apiKey: "xxxx",
+  federationUrl: "http://xxx.xxx.macrometa.io",
+  // this is an optional parameter to be passed when you want to use the endpoint as it is
+  absoluetPath:true
 });
 ```
 
 ### Initializing macrometa dynamodb
 
 ```js
-import { DynamoDB } from "mmdynamo";
+import  Client  from "mmdynamo";
 
-const dcName = "test.macrometa.io";
-const host = "https://api-" + dcName;
+const agent = "fetch";
+const federationUrl = "http://xxx.xxx.macrometa.io";
 const apiKey = "xxxxxxxxxxxxxxxxxxxxxx";
-/*
-   If you have a JWT token
-   
-   const JWT = "xxxxxxxxxxxxxxxxxxxxxx";
-   */
-
-const service = "dynamodb";
-const region = "us-east-1";
-const endpoint = host + "/_api/dynamo";
-// secretAccessKey is a required parameter for aws-sdk we recommend you to pass "c8"
-const secretAccessKey = "c8";
-const accessKeyId = "apikey " + apiKey;
-/*
-   If you are using JWT token use the below instead
-   
-   const accessKeyId = "bearer " + JWT;
-   */
-
-const client = new DynamoDB({
-  region,
-  endpoint,
-  accessKeyId,
-  secretAccessKey,
+// this is an optional parameter to be passed when you want to use the endpoint as it is
+const absolutePath=true
+
+
+const client = new Client({
+  agent,
+  federationUrl,
+  apiKey,
+  absolutePath,
 });
 ```
 
 ### Create Table
 
 ```js
-client
-  .createTable({...},(err,data)=>{
-    console.log(err,data)
-  })
+      const response = await client.createTable()
+      console.log(response)
 ```
 
-### Describe Table
 
-```js
-client
-  .describeTable({...},(err,data)=>{
-    console.log(err,data)
-  })
-```
 
-### Describe TimeToLive
 
-```js
-client
-  .describeTimeToLive({...},(err,data)=>{
-    console.log(err,data)
-  })
-```
 
 ### List Tables
 
 ```js
-client.listTables().then((res) => {
-  return res.json();
-});
+      const response = await client.listTables()
+      console.log(response)
```

Comment:
correct this indentation for all instances. i have added image please check.

Reply:
ok sure checking

Reply:
done